client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-rule-edit.html

```diff
@@ -324,7 +324,8 @@ <h2 data-translate="{{ $ctrl.rule.status === 'DRAFT' ? 'telephony_number_feature
             <section data-ng-switch-when="actions">
                 <div class="section-group">
                     <div class="section-item"
-                         data-ng-repeat="action in $ctrl.availableActions | orderBy:'label' track by $index">
+                         data-ng-repeat="action in $ctrl.availableActions | orderBy:'label' track by $index"
+                         data-ng-if="(action.value === 'ivr' && $ctrl.ovhPabx.featureType !== 'cloudHunting') || action.value !== 'ivr'">
```

Comment:
Please make the test in the controller :-)
https://github.com/ovh-ux/ovh-manager-telecom/blob/f84d7e21675bcf46e7f24c885222cfe4d3ce6711/client/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/telephony-group-number-feature-ovh-pabx-dialplan-extension-rule-edit.controller.js#L252-L258